README.md

```diff
@@ -26,7 +26,14 @@ Windows Platform tools are required. Install them with:
 npm install --global --production windows-build-tools
 ```
 
+
+
+#### 'vue-cli-service' is not recognized
+
+Please see issue 122 (https://github.com/vcync/modv-3/issues/122) on how to resolve this.
```

Comment:
Please change the link https://github.com/vcync/modv-3/issues/122#issuecomment-640100114 to point directly to the comment that fixes the problem.

Reply:
Sure thing, updated!